controller/src/main/java/org/jboss/as/controller/operations/global/AbstractCollectionHandler.java

```diff
@@ -81,28 +80,60 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
             attributeName = extractAttributeName(attributeName);
         }
         final AttributeAccess attributeAccess = context.getResourceRegistration().getAttributeAccess(PathAddress.EMPTY_ADDRESS, attributeName);
-        if (attributeAccess == null) {
-            throw new OperationFailedException(ControllerLogger.ROOT_LOGGER.unknownAttribute(attributeName));
-        } else if (requiredReadWriteAccess && attributeAccess.getAccessType() != AttributeAccess.AccessType.READ_WRITE) {
-            throw new OperationFailedException(ControllerLogger.ROOT_LOGGER.attributeNotWritable(attributeName));
-        }
```

Comment:
Note to self: I believe this is just pruning because ReadAttributeHandler or WriteAttributeHandler will have already done these checks before this handler ever gets called.